src/components/Calendar/src/Calendar.vue

```diff
@@ -0,0 +1,455 @@
+<template>
+	<div
+		:class="$s.Calendar"
+		v-bind="$attrs"
+		v-on="$listeners"
+	>
+		<div :class="$s.CalendarHeader">
+			<m-button
+				:disabled="shouldDisableHeaderButton(-1)"
+				:color="headerButtonColor"
+				size="small"
+				variant="primary"
+				tabindex="-1"
+				@click="incMonth(-1)"
+			>
+				<chevron-left
+					:class="$s.CalendarHeaderButtonIcon"
+				/>
+			</m-button>
+
+			<span :class="$s.CalendarHeaderTitle">
+				{{ monthName }}
+			</span>
+
+			<m-button
+				:disabled="shouldDisableHeaderButton(1)"
+				size="small"
+				variant="primary"
+				tabindex="-1"
+				color="#f2f2f2"
+				@click="incMonth(1)"
+			>
+				<chevron-right
+					:class="$s.CalendarHeaderButtonIcon"
+				/>
+			</m-button>
+		</div>
+		<table :class="$s.CalendarTable">
+			<thead>
+				<tr>
+					<th
+						v-for="day in weekdays"
+						:key="`day-${day}`"
+						:class="$s.DateHeaderCell"
+					>
+						{{ day }}
+					</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr
+					v-for="(week, wi) in weeks"
+					:key="`week-${wi}`"
+				>
+					<td
+						v-for="(date, di) in week"
+						:key="`date-${wi-di}`"
+						:class="$s.DateCell"
+						@click.prevent="emitDate(date)"
+					>
+						<button
+							v-if="date"
+							:class="[$s.DateCellButton, {
+								[$s.selected]: isDateSelected(date),
+								[$s.disabled]: isDateDisabled(date),
+								[$s.today]: isToday(date)
+							}]"
+							type="button"
+							tabindex="-1"
+						>
+							{{ date }}
+						</button>
+					</td>
+				</tr>
+			</tbody>
+		</table>
+	</div>
+</template>
+
+<script>
+import {
+	startOfWeek,
+	startOfMonth,
+	eachDayOfInterval,
+	getDay,
+	lastDayOfMonth,
+	addDays,
+	format,
+	isToday,
+	isAfter,
+	isSameDay,
+	isSameMonth,
+	isSameYear,
+	addMonths,
+	parse,
+} from 'date-fns';
```

Comment:
This seems to be bundling in all of date-fns, which explains the `657.9 kB` size.

(The total size is much larger due to source maps)
<img width="495" alt="Screen Shot 2021-03-24 at 3 33 03 PM" src="https://user-images.githubusercontent.com/1075694/112372575-5d2b7e00-8cb6-11eb-86c2-7dcb82eec966.png">


We should consider upgrading to Webpack 5 for better tree-shaking. In the mean time, you can either:
1. Import each method directly by path. eg. `import format from 'date-fns/format'`;
2. Use [`babel-plugin-date-fns`](https://github.com/date-fns/babel-plugin-date-fns) to automatically convert to the above syntax